packages/just-scripts/package.json

```diff
@@ -6,31 +6,33 @@
   "author": "",
   "license": "MIT",
   "main": "./lib/index.js",
-  "dependencies": {},
-  "devDependencies": {
+  "dependencies": {
+    "chalk": "^2.4.1",
```

Comment:
I purposefully made these devDeps and made this into a webpack bundle so install time is nil for this package. If there's a package we needed to expose somehow, then we should do it thru webpack.config.js externals.

Reply:
Ah okay. Reverted that part.